### DISCUSSION TITLE
Can't boot into UMAF

### DISCUSSION THREAD

**Discussion by trantuananh1996** [DavidS95/Smokeless_UMAF#21] [General]
I've try 2 USB, FAT32, extract file to root folder. And it won't boot into UMAF. It boot into OS
I'm using lenovo thinkbook 14p, with R7 5800H
Any advice? Thanks

**Comment by trantuananh1996** [comment]
Well I figured it out. Format with windows doesn't work. Using rufus, it can boot now

**Comment by evlo** [comment]
and here i thought it is issue with uefi/legacy boot

so you created NON BOOTABLE in rufus or what did you use exactly?
